GoogleChromeLabs/ps-analysis-tool
# Fix: memory usage by PSAT UI

## Description
In our effort to increase the performance of the extension, we have added a PR to reduce the memory usage of the extension.

## Relevant Technical Choices
- Data from the service worker will not be emitted if there isn't any new data to display.
- Deep object diff has been replaced by loadash is equal.

## Testing Instructions
- Clone this branch.
- In the terminal run `npm i && npm run build:ext`.
- Open any website the extension should not break.

## Additional Information:



## Screenshot/Screencast



---

## Checklist



~~[ ] This code is covered by unit tests to verify that it works as intended.~~
